packages/drivers/local-driver/src/localDocumentStorageService.ts

```diff
@@ -82,6 +94,13 @@ export class LocalDocumentStorageService implements IDocumentStorageService {
 		summary: ISummaryTree,
 		context: ISummaryContext,
 	): Promise<string> {
+		if (context.referenceSequenceNumber === 0) {
+			await createDocument(
```

Comment:
the error should be here right before we need them

Reply:
this will then only break in the new feature case, existing usage will continue to work